eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -165,6 +170,47 @@ stages:
                           PRLabels: "auto-merge"
                           CloseAfterOpenForTesting: '${{ parameters.TestPipeline }}'
 
+                      - task: PowerShell@2
```

Comment:
Given you are only using this in another PS step you don't have to separate the steps you can just call the script in the pwsh inline script and get the results without needing to use the VSOVariable.

Another advantage of doing that inline is that you can call it twice instead of needing to hard-code a fallback owner. You can call the script again with the TargetDirectory "sdk" to get the default list in case there isn't any found.